.github/workflows/validate_cwls.yml

```diff
@@ -0,0 +1,31 @@
+name: Validate CWL Files
+
+on:
+  push:
+    branches: develop
```

Comment:
you might add for the master branch here too (if we plan on ever using the master branch)

Reply:
yep makes sense 👍